mitodl/ocw-to-hugo
# store course metadata in hugo data templates

#### Pre-Flight checklist


#### What are the relevant tickets?
Part of https://github.com/mitodl/hugo-course-publisher/issues/326

#### What's this PR do?
This PR stores course level metadata in json files located at `site/data` and alters the root of `ocw-to-hugo`'s expected output to be `hugo-course-publisher`'s `site` folder.

#### How should this be manually tested?
Generate markdown / JSON using this branch and load it into `hugo-course-publisher` in [this](https://github.com/mitodl/hugo-course-publisher/pull/419) PR branch.  Keep in mind that the expected root of the output directory is now the `site` folder inside `hugo-course-publisher`.  As an example, this is what I ran:

```sh
node . -i ~/Code/open-learning-course-data -o ~/Code/hugo-course-publisher/site/
```
